cmd/version.go

```diff
@@ -50,6 +51,11 @@ func newVersionCmd(homedir, getIstioVersion string) *cobra.Command {
 
 			logger.Infof("getistio version: %s\nactive istioctl: %s\n", getIstioVersion, cur.ToString())
 
+			if _, err := util.GetK8sClient(); err != nil {
```

Comment:
what if, for example, the current context is valid in its format but it's pointing the wrong cluster?